site/src/Librecores/ProjectRepoBundle/Resources/views/Project/settings.html.twig

```diff
@@ -6,6 +6,35 @@
       "assets/scss/trumbowyg.scss"
   %}
     <link rel="stylesheet" href="{{ asset_url }}" />
+    <style type="text/css">
```

Comment:
Please move this CSS into our standard stylesheet (librecores.css) and use the coding style there (e.g. the opening brace `{` on the same line as the selector.

Reply:
Moved it to the librecores.css stylesheet